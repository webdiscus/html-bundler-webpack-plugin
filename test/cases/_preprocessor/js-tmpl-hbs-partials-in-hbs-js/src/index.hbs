<!DOCTYPE html>
<html>
  <head>
    <title>{{ title }}</title>
    <script src="app.js" defer="defer"></script>
  </head>
  <body>
    {{> 'header'}}
    {{!-- the partial (name containing '/') used in both a Handlebars template and a JS template --}}
    {{> 'header/subheader' subheaderTitle="this partial used in HBS-template"}}
    {{!-- the partial used in both a Handlebars template and a JS template --}}
    {{> person name="Heisenberg" age="50"}}
    <div id="app"></div>
  </body>
</html>