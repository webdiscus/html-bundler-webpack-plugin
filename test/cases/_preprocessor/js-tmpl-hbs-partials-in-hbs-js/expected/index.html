<!DOCTYPE html>
<html>
  <head>
    <title>My Title</title>
    <script src="app.js" defer="defer"></script>
  </head>
  <body>
    <h1 class="header">HEADER</h1>    <h3>Sub header: this partial used in HBS-template</h3>    <div>Person: Heisenberg, 50</div>
    <div id="app"></div>
  </body>
</html>